src/math/LinearCombinationKernel.cc

```diff
@@ -107,6 +107,24 @@ std::shared_ptr<Kernel> LinearCombinationKernel::clone() const {
     return retPtr;
 }
 
+std::shared_ptr<Kernel> LinearCombinationKernel::resized(int width, int height) const {
+    KernelList kernelList;
+
+    for (KernelList::const_iterator kIter = _kernelList.begin(), kEnd = _kernelList.end(); kIter != kEnd;
+         ++kIter) {
+        std::shared_ptr<Kernel> basisKernelPtr = (*kIter)->resized(width, height);
+        kernelList.push_back(basisKernelPtr);
```

Comment:
I don't have a feeling either way, I was just wondering if that is why you chose to write it all out.